monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -5010,8 +5010,8 @@ object Observable extends ObservableDeprecatedBuilders {
     *  - ensures that the input stream is closed on completion,
     *    failure or cancellation
     *
-    * @param in is the `Task[InputStream]` generator to convert into an observable
-    * @param chunkSize is the maximum length of the emitted arrays of bytes
+    * @param in the `Task[InputStream]` generator to convert into an observable
+    * @param chunkSize the maximum length of the emitted arrays of bytes, must be positive
```

Comment:
I patterned the chunkSize restriction scaladoc to Observable.range, where it didn't really specify that an IllegalArgumentException can be thrown, but just mentioned in the scaladoc what is valid.